Sonifcation/Function.cs

```diff
@@ -19,15 +21,45 @@ public partial class Function : Node {
 	/// Stores the AST of the Function.
 	/// </summary>
 	public IFunctionAST FunctionAST {get; set;}
+
+	/// <summary>
+	/// If <c> true </c>, Function is a constant function (i.e., y = 1, y = 3, y = 6942).
+	/// </summary>
+	public bool IsConstant {get; set;}
```

Comment:
Why do you need to explicitly handle the case where the function is constant?

Reply:
```
// Find the full range of values that the function can take.
		double minVal = FunctionTable.Min();
		double maxVal = FunctionTable.Max();
		double functionRange = maxVal - minVal;
		
		// Map the Functions values to note numbers
		foreach(double value in FunctionTable) {
			int noteNumber;
			if(IsConstant) noteNumber = (int) Math.Abs(FunctionTable[0]) % noteRange;
			else {
				double normalizedValue = (value - minVal) / functionRange;
				noteNumber = noteNumStart + (int) (normalizedValue * noteRange);
			}
			noteSequence.Add(noteNumber);
		}
```

So, looking at `functionRange`, this is the allowable range of function outputs calculated as the difference between the maximum output and maximum input from when they were calculated using `FillFunctionTable()`. In a constant function, this ends up being 0 and that carries into the calculation for `noteNumber` and makes it infinity thus throwing the access to `Notes` in `Frequencies.cs` that holds the notes to play out of bounds.